modules/internal/ChapelStandard.chpl

```diff
@@ -56,6 +56,7 @@ module ChapelStandard {
   use MemTracking;
   use ChapelUtil;
   use ChapelDynDispHack;
+  use ChapelEnv;
```

Comment:
This file roughly determines the order of module initialization. Since you use ChapelEnv in ChapelBase, it may as well go to the above ChapelBase here.